### DISCUSSION TITLE
파이썬 라이브러리 이름을 추천 받습니다.

### DISCUSSION THREAD

**Discussion by hahnlee** [hahnlee/hwp-rs#38] [자유게시판]
라이브러리를 개발할때, 파이썬 라이브러리 이름을 hwp나 pyhwp를 원했습니다. 이미 사용중인 이름이라 개발중에는 hwppy라는 이름을 사용했었는데요. pypi의 정책상 다른 라이브러리와 너무 유사한 이름은 사용할 수 없어 libhwp로 바꾸고 공개하였습니다.

그런데 과거에 libhwp로 사용한 라이브러리가 있어 여러모로 혼란스러울것 같아 이름을 바꾸는걸 고려중인데 적당한 이름이 생각나지 않네요.

**Comment by mrchypark** [comment]
galapygos 는 어떤가요? 갈라파고스를 타파하고자 하는 역설적인 이름입니다.

**Reply by Leo7654** [reply]
ㅋㅋㅋ 갈라파고스에 booby라는 새가 있는데 멍청하단 뜻이네요.
